src/aead/aes.rs

```diff
@@ -19,6 +19,7 @@ use super::{
 use crate::{bits::BitLength, cpu, endian::*, error, polyfill};
 use libc::size_t;
 
+#[derive(Clone)]
```

Comment:
Please modify aead_test.rs to verify that the clone of a key produces the same results as the original key.

Reply:
Done.